.gitignore

```diff
@@ -1,6 +1,4 @@
 .astro/
 dist
-node_modules
```

Comment:
We shouldn't remove `node_modules`.

Reply:
Why? Biome ignores it already

Reply:
But this is for `git` to ignore it, isn't it?

Reply:
Lol true 😅